datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/jmx.yaml

```
@@ -104,6 +104,12 @@
     will give up on that connection and retry.
   value:
     example: 15000
-    type: number
+    type: number 
+
+- name: collect_default_jvm_metrics
+  description: Allows disabling the collection of default JVM metrics
```

```suggestion
  description: Enables the collection of default JVM metrics
```
Small nit but feel like this wording would be more clear since it is disabled by default.

---

Sorry if this was not clear! Actually this defaults to true. Perhaps I should call that out in the doc?
For context - the existing behavior is to collect default JVM metrics but I wanted to add a flag to explicitly disable that.

I thought it may be misleading to have a `disable_collect_default_jvm_metrics: true` because that logic feels backwards to me.